src/index.js

```diff
@@ -37,6 +38,7 @@ class Store extends BaseStore {
     this.endpoint = process.env.GHOST_STORAGE_ADAPTER_S3_ENDPOINT || endpoint || ''
     this.serverSideEncryption = process.env.GHOST_STORAGE_ADAPTER_S3_SSE || serverSideEncryption || ''
     this.s3ForcePathStyle = Boolean(process.env.GHOST_STORAGE_ADAPTER_S3_FORCE_PATH_STYLE) || Boolean(forcePathStyle) || false
+    this.signatureVersion = process.env.GHOST_STORAGE_ADAPTER_S3_SIGNATURE_VERSION || signatureVersion || 'v4'
```

Comment:
@AMD-NICK @eofs thanks for your help with this! My only concern here is what the default `signatureVersion` should be. Here it's `v4`, but will this not break for existing users who upgrade and aren't setting `signatureVersion`? Or will `v4` work okay for users on `v2` and `v3` too?

Reply:
I think so. I'm new in AWS, but I've seen that v4 supports in all regions